src/tfr_executive/src/teleop_action_server.cpp

```diff
@@ -141,12 +159,21 @@ class TeleopExecutive
                 ROS_INFO("Teleop Action Server: Command Revieved, DUMP");
                 //TODO hook up dumping when ready
             }
-            else if (code == tfr_utilities::TeleopCode::RESET)
+            else if (code == tfr_utilities::TeleopCode::RESET_DUMPING)
             {
-                ROS_INFO("Teleop Action Server: Command Revieved, RESET");
+                ROS_INFO("Teleop Action Server: Command Revieved, RESET_DUMPING");
+                //all zeros by default
+                drivebase.publish(move_cmd);
+
                 //TODO hook up reset when ready
             }
-            else
+            else if (code == tfr_utilities::TeleopCode::RESET_STARTING)
+            {
+                ROS_INFO("Teleop Action Server: Command Revieved, RESET_STARTING");
+                //all zeros by default
+                drivebase.publish(move_cmd);
+                //TODO hook up reset when ready
+            }else
             {
                 ROS_WARN("Teleop Action Server: UNRECOGNIZED COMMAND");
```

Comment:
We return a `succeeded` state on unrecognized commands, even though we take no action.  Should probably set `failure` here and return.

Reply:
The state we want to signal that behavior is called `aborted` but I know what you mean, sure